sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/Scenario/IssueTests.cs

```diff
@@ -158,7 +158,11 @@ public async Task CRUD()
             // Load a filestream and put its content into the byte[]
             using (FileStream fs = fileInfo.OpenRead())
             {
+#if NET6_0_OR_GREATER
```

Comment:
[CA2022: Avoid inexact read with Stream.Read](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2022) ReadAsync may return fewer bytes than requested which can result in unreliable code. ReadExactlyAsync should be used instead but it's only available for .NET6+. Since this is a test case, just leave the net462 version as-is.